css/style.css

```diff
@@ -356,3 +370,92 @@ div .taggs {
   padding: 0 0.5rem;
   margin: 1em;
 }
+
+/* Contact page */
+.contact {
+  width: 100%;
+  height: 740px;
+  background: #6070ff;
+  background-image: url(/images/contact/layers.svg);
+  background-size: 600px 750px;
+  background-repeat: no-repeat;
+  background-position-x: -70px;
+  background-position-y: 10px;
+  border-top-left-radius: 6rem;
+  padding-bottom: 4rem;
+}
+
+.contact-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  margin-top: 80px;
+  padding-top: 2.5rem;
+  margin-bottom: 0;
+}
+
+.contact-content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  text-align: center;
+  color: #ebebff;
+  padding: 0 1.5rem;
+}
+
+.my-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1.5rem;
+  gap: 1.3rem;
+}
+
+.inputField {
+  padding: 0.9rem;
+  border-radius: 10px;
+  border: none;
+  font-style: normal;
+
+  /* font-family: 'Poppins', sans-serif; */
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.inputField:hover {
+  transform: translate(5px);
+}
+
+.textareaField {
+  border-radius: 10px;
+  padding: 0.9rem;
+  font-style: normal;
+
+  /* font-family: 'Poppins', sans-serif; */
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  height: 170px;
+}
+
+.textareaField:hover {
+  transform: translate(5px);
+}
+
+.my-button {
+  align-items: center;
+  padding: 12px;
+  background: #fff;
+  border-radius: 8px;
+  color: #6070ff;
+  width: 50%;
+  border: none;
+
+  /* font-family: 'Poppins', sans-serif; */
+  font-size: 17px;
+  font-weight: 500;
+  cursor: pointer;
+}
```

Comment:
- [ ] You are almost there @ezraermy , there's a few more tweaks to perfect. It appears you missed the suggestion of the previous reviewers about implementing all button interactivity for your `Get in touch` button like every other `button` on your page. Please check the full recommendation [here](https://github.com/ezraermy/ezra_portfolio/pull/4#discussion_r1072800877). Implementing this will really make your page more appealing to the eye aesthetically, besides it also improves user experience. Here's a tip on keeping your code `DRY` with this implementation; I see that you have classes declared on other buttons with different styling for their `hover`, `active` and `enabled` states, I think you can just add one of those classes to your `button` tag like 👇 :
```
<button type="submit" class="my-button resume">Get In Touch</button>
```

Reply:
HI @AbayomiOlaoye. Thanks for the feedback. Given that the contact background itself is blue, making the 'get in touch' button interaction hover and pressed blue would make it invisible to watch. I just thinking.

Reply:
But in the end, a developer must stick to project requirements. Remember that you're not the user, so we have to build according to specifications or suite the need of users, at the very least, we can offer professional advice or consultation - but this is not applicable in this case. Kindly stick to the Figma design @ezraermy 👏👏🙏👋